corehq/apps/app_manager/tests/test_suite_resource_overrides.py

```diff
@@ -0,0 +1,59 @@
+from django.test import TestCase
+
+from corehq.apps.app_manager.exceptions import ResourceOverrideError
+from corehq.apps.app_manager.suite_xml.post_process.resources import add_xform_overrides
+from corehq.apps.app_manager.tests.app_factory import AppFactory
+from corehq.apps.app_manager.tests.util import TestXmlMixin
+
+
+class SuiteResourceOverridesTest(TestCase, TestXmlMixin):
+    file_path = ('data', 'suite')
+
+    @classmethod
+    def setUp(self):
+        super().setUpClass()
```

Comment:
I don't know what this is going to do. Should this be setup or setupclass?

Reply:
92682c18142